parm/config/gfs/config.resources

```diff
@@ -889,8 +889,8 @@ case ${step} in
     threads_per_task=1
     walltime_gdas="03:00:00"
     walltime_gfs="06:00:00"
-    ntasks=4
-    tasks_per_node=4
+    ntasks=1
+    tasks_per_node=1
```

Comment:
This wasn't what I had in mind when I said "serial", and I don't think fixes anything. The tasks need to be serial, not the job execution. Marking a metatask as serial is a capability I'm trying to add in #2795. Recommend taking this out and only addressing the name issue.

Reply:
@WalterKolczynski-NOAA I understand now by what you mean by serial.  That said, there is an intermittent bug in METplus when running it with more than 1 task, and this does fix that issue.  See [this comment](https://github.com/NOAA-EMC/global-workflow/pull/2791#issuecomment-2260424914) for more details.

Reply:
That's the one I was referring to as well, I just misunderstood the problem I guess.